auth/src/main/java/com/firebase/ui/auth/ui/idp/AuthMethodPickerActivity.java

```diff
@@ -106,6 +108,11 @@ protected void onFailure(@NonNull Exception e) {
                 }
             }
         });
+
+        TextView termsText = findViewById(R.id.main_tos_and_pp);
+        PrivacyDisclosureUtils.setupTermsOfServiceAndPrivacyPolicyText(AuthMethodPickerActivity.this,
```

Comment:
nit: do you need `AuthMethodPickerActivity.this` or could it be just `this`?

Reply:
I'll change to this